### DISCUSSION TITLE
取模计算

### DISCUSSION THREAD

**Discussion by kuangkung** [he3-app/he3-feedback#45] [Ideas]
可以添加一个取模的计算工具，参考[取模运算计算器]:https://www.99cankao.com/algebra/modulo-calculator.php

**Comment by kuangkung** [comment]
求模运算与求余运算不同。“模”是“Mod”的音译，模运算多应用于程序编写中。 Mod的含义为求余。模运算在数论和程序设计中都有着广泛的应用，从奇偶数的判别到素数的判别，从模幂运算到最大公约数的求法，从孙子问题到凯撒密码问题，无不充斥着模运算的身影。虽然很多数论教材上对模运算都有一定的介绍，但多数都是以纯理论为主，对于模运算在程序设计中的应用涉及不多。

例如11 Mod 2，值为1

上述模运算多用于程序编写，举一例来说明模运算的原理：

Turbo Pascal对mod的解释是这样的：

A Mod B=A-(A div B) * B （div含义为整除）

**Comment by hongchaodeng** [comment]
感谢你的反馈，我们先记一下哈。
